### ISSUE TITLE
Windows sleep causes crash, can't uninstall

### ISSUE THREAD

**Issue by naasking** [zerotier/ZeroTierOne#387]
Ongoing issues from a prior attempt to bridge connections in Windows which I reported here:

https://www.zerotier.com/community/topic/54/bridging-zerotier-and-ethernet-causes-persistent-failure/10

I've left ZeroTier uninstalled since then, but had to reinstall for some remote management purposes yesterday. Here are the resulting problems:
1. Trying to sleep shuts down the machine entirely, it doesn't enter sleep mode.
2. Can't uninstall ZeroTier in normal windows mode, it just hangs right at the start. I had to reboot into safe mode, manually start the windows installer service, then uninstall ZeroTier. Computer now properly sleeps.

If there is any data you need from my machine to analyze this issue, please let me know. I've tried the bridging described in that forum post on Windows 7 and Windows 10 on the same hardware, and both of them failed and caused problems on the machine afterwards. Windows 7 got borked which is why I upgraded to Windows 10 at the time, only to have the issue recur.

**Comment by adamierymenko** [comment]
Was there any other network-related or virtualization software installed? HyperV, Parallels, OpenVPN, Hamachi, etc.?

I wonder if this is in any way related to #308 or #357 -- neither of which we have been able to reproduce here. :(

**Comment by naasking** [comment]
I did not have Hyper-V on Windows 7 when I first encountered the issue. I do have it on Windows 10 now and when I tried the bridge in the forum post after a clean install. All the information about my setup is documented in that series of forum posts. The behaviour described in this ticket is the only new info since it recently happened, so clearly ZeroTier has borked itself or some obscure part of my system when it's running.

One thing I didn't mention in the forum post, is that I have only one Realtek LAN adapter for network connectivity, and that's what I tried to bridge with, so perhaps that will help you reproduce the problem. Perhaps that ridiculous attempt is why you haven't been able to reproduce it, because why would anyone do that? ;-)

I did the same bridging in Debian from within Hyper-V and while I couldn't ping, I could remove the bridge and everything resumed working as expected. It's specifically bridging with your only LAN adapter that seems to cause the trouble on Windows, though perhaps it's specific to Realtek, or even my specific Realtek chip.
